src/main/scala/zio/ftp/FtpError.scala

```diff
@@ -0,0 +1,12 @@
+package zio.ftp
```

Comment:
I'd call this file `errors.scala` since you have more than one top-level class in it.